doc/en/Tutorial - Code a module.rst

```diff
@@ -1599,8 +1604,8 @@ it's a convention): ::
 
 Some explanations :
 
-- The attribute ``permission`` is a string using Django's conventions for
-  permissions, with a shape : 'APP-ACTION'.
+- The attribute ``permission`` is a list of strings using Django's conventions for
+  permissions, with a shape : ['APP-ACTION', …].
```

Comment:
-"The attribute ``permissionS``"
- As seen in my previous comment it can be a simple string too.